### [CVE-2024-53432](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-53432)
![](https://img.shields.io/static/v1?label=Product&message=n%2Fa&color=blue)
![](https://img.shields.io/static/v1?label=Version&message=1.14.1%20&color=brightgreen)
![](https://img.shields.io/static/v1?label=Version&message=n%2Fa%20&color=brightgreen)
![](https://img.shields.io/static/v1?label=Vulnerability&message=n%2Fa&color=brightgreen)

### Description

While parsing certain malformed PLY files, PCL version 1.14.1 crashes due to an uncaught std::out_of_range exception in PCLPointCloud2::at. This issue could potentially be exploited to cause a denial-of-service (DoS) attack when processing untrusted PLY files.

### POC

#### Reference
- https://github.com/PointCloudLibrary/pcl/issues/6162

#### Github
No PoCs found on GitHub currently.

